docs/DeveloperGuide.adoc

```diff
@@ -867,6 +876,15 @@ and still be able to filter by a combination of the different arguments.
 
 Eventually we may decide to use a combination of both alternatives as it is possible that different arguments are easier to
 implement using different methods.
+
+==== Proposed Extension
+In the near future, we plan to enhance the filtering feature. We want to improve the list command to take in tags as a
+possible argument, and as such, allow for the possible filtering by tags as well. The design considerations mentioned
+earlier was to facilitate this proposed extension, since we would require the use of a `**Predicate**` which will be
+created using the tag in the argument as well. By doing so, we reduce the difficulty of enhance the filtering feature in
+the future. This would benefit the simplicity of the command and further enhance the user experience, for them to be able
+to filter by a combination of both months and tags.
+
 // end::Filtering transactions[]
 
 //tag::dataencryption[]
```

Comment:
Remember to remove the test case stated in Appendix F (F.7) part d.

Reply:
roger